lib/plugins/aws/deploy/lib/uploadArtifacts.js

```diff
@@ -119,42 +141,56 @@ module.exports = {
         })
         .filter(Boolean)
     );
+  },
 
+  getLayerArtifactFilePaths() {
     const layerNames = this.serverless.service.getAllLayers();
-    artifactFilePaths = artifactFilePaths.concat(
-      layerNames
-        .map((name) => {
-          const layerObject = this.serverless.service.getLayer(name);
-          if (layerObject.artifactAlreadyUploaded) {
-            this.serverless.cli.log(`Skip uploading ${name}`);
-            return null;
-          }
-          return getLambdaLayerArtifactPath(
-            this.packagePath,
-            name,
-            this.provider.serverless.service,
-            this.provider.naming
-          );
-        })
-        .filter(Boolean)
-    );
+    return layerNames
+      .map((name) => {
+        const layerObject = this.serverless.service.getLayer(name);
+        if (layerObject.artifactAlreadyUploaded) {
+          legacy.log(`Skip uploading ${name}`);
```

Comment:
Is an intention to never show this message with new logs (even in verbose mode) ?

Reply:
It would make sense to me to show this in verbose mode (but not in normal mode because it would be too fast to display in the progress anyway, and it doesn't look like critical information)

Reply:
no, in my opinion it should be shown in verbose mode that I wanted to address in separate PR, I'm going to revert the change here and cover both switch to legacy.log and new log in verbose mode